test/ut/sdk/test_compression_utils.py

```diff
@@ -138,5 +139,37 @@ def test_mask_conflict(self):
                         assert b_index1 == b_index2
 
 
+    def test_flops_params(self):
+        class Model1(nn.Module):
+            def __init__(self):
+                super(Model1, self).__init__()
+                self.conv = nn.Conv2d(3, 5, 1, 1)
+
+            def forward(self, x):
+                return self.conv(x)
+
+        class Model2(nn.Module):
+            def __init__(self):
+                super(Model2, self).__init__()
+                self.conv = nn.Conv2d(3, 5, 1, 1)
+                self.conv2 = nn.Conv2d(5, 5, 1, 1)
+
+            def forward(self, x):
+                x = self.conv(x)
+                for _ in range(5):
+                    x = self.conv2(x)
+                return x
```

Comment:
better to add linear op. and some ops in `OTHER_OPS`.

Reply:
added

Reply:
looks good